queries.sql

```diff
@@ -17,3 +17,20 @@ SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.40 AND 17.30;
 
+
```

Comment:
- [ ] Please, implement these project requirements:


## Explanation

I can't find your implementation of these project requirements.  We need to add transactions and verify them.

## Hints

- A transaction is a code block that begins with a `BEGIN;` statement and ends with a `COMMIT;` or `ROLLBACK;` statement.
- The `COMMIT` statement applies the changes, and the `ROLLBACK` statement cancels them.
- You need to verify each step with a `SELECT` statement to verify that each change has been made.
- The `SELECT` statements need to exist after each rollback, commit, update, or delete statement.
- Example:

```SQL
BEGIN;

   UPDATE animals SET species = 'unspecified'; 

   SELECT species from animals; -- VERIFY EACH CHANGE LIKE THIS!
```

Now that you know how to do it, please kindly implement it on your code to meet this requirement 💪